protocol/oracle-manager/contracts/nodes/UniswapNode.sol

```diff
@@ -16,12 +18,21 @@ library UniswapNode {
     using SafeCastU56 for uint56;
     using SafeCastU32 for uint32;
     using SafeCastI56 for int56;
+    using SafeCastI256 for int256;
+
+    using DecimalMath for int256;
+
+    uint8 public constant PRECISION = 18;
```

Comment:
I believedata for what precision we are targeting is also stored in the `DecimalMath` library